ProcessMaker/screen-builder
# Feature/FOUR-18113: STORY Extract values from a Record List

## How to Test
- Login PM4
- Go to Designer->new Screen
- Drag Record List control
- In Configuration Select "Collection" from dropdown
- Select a pre loaded Collection
- Go to Data Selection and choose "Single field of Record" or "Single Record" or "Multiple Records"
- Go to Columns and select some columns to be displayed
- Go to Preview and check Radio buttons or Checkboxes depending Data Selection option

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-18113


## Code Review Checklist
ci:next
